.github/workflows/diagram-nightly.yml

```
@@ -17,7 +17,7 @@ jobs:
       contents: read
       issues: write
     runs-on: ubuntu-latest
-    timeout-minutes: 30
+    if: ${{ false }} # Full-corpus render/regression gates are disabled.
```

actionlint (1.7.12)

[error] constant expression "false" in condition. remove the if: section

(if-cond)